From 91dadb9eec7f3fc7b8499a0cf24d55569d18a406 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Wed, 8 Nov 2017 18:36:00 +0800
Subject: [PATCH] ARM: dts: rockchip: fixes the emmc error on rk3036 SoCs

As the emmc is supplyed power by vcc_io, that's 3.3v voltage.
the default 1.8v volatge will cause the emmc error. as the following:

[   17.096082 ] mmcblk1: error -115 sending stop command, original cmd
response 0x900, card status 0xb00
[   17.127022 ] mmcblk1: error -110 transferring data, sector 664720, nr
72, cmd response 0x900, card status 0xb00

Remove the mmc-ddr-1_8v to keep the default the 3.3v voltage.

Change-Id: I9e2539d63fd93e72d9febbb311fbd686c5a11d09
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm/boot/dts/rk3036.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3036.dtsi b/arch/arm/boot/dts/rk3036.dtsi
index df9ab4298101..89b300c21643 100644
--- a/arch/arm/boot/dts/rk3036.dtsi
+++ b/arch/arm/boot/dts/rk3036.dtsi
@@ -434,7 +434,6 @@
 		dmas = <&pdma 12>;
 		dma-names = "rx-tx";
 		fifo-depth = <0x100>;
-		mmc-ddr-1_8v;
 		non-removable;
 		num-slots = <1>;
 		supports-emmc;
-- 
2.35.3

